<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1775</width>
    <height>976</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>20</y>
      <width>171</width>
      <height>811</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Vel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="angle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>angle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yaw_rate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>yaw_rate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenFile">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cursor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>光标捕捉</string>
     </property>
    </widget>
    <widget class="QSlider" name="xAxis_zoom">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>185</y>
       <width>18</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="yAxis_zoom">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>185</y>
       <width>18</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="showMovie">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>710</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>     开启动画</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Analyse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>640</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>      Analyse</string>
     </property>
    </widget>
    <widget class="QSlider" name="move_speed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>815</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>780</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Speed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Normal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>      Normal</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>1401</width>
      <height>821</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1775</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action17863110617_qq_com"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action17863110617_qq_com">
   <property name="text">
    <string>17863110617@qq.com</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
